src/main/java/io/codelex/flightplanner/flights/controller/AdminController.java

```diff
@@ -0,0 +1,35 @@
+package io.codelex.flightplanner.flights.controller;
+
+import io.codelex.flightplanner.flights.domain.Flight;
+import io.codelex.flightplanner.flights.dto.FlightRequest;
+import io.codelex.flightplanner.flights.service.FlightService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.text.ParseException;
+
+@RestController
+@RequestMapping("/admin-api")
+public class AdminController {
+
+    private FlightService service;
+
+    public AdminController(FlightService service) {
+        this.service = service;
+    }
+
+    @PutMapping("/flights")
+    public ResponseEntity<Flight> addFlight(@RequestBody FlightRequest flightRequest) throws ParseException {
```

Comment:
Man liekas šitam vajadzētu strādāt arī newrapojot to Flight iekš ResponseEntity, bet ja šīs metode vienkārši atgrieztu Flight. Tas pats ar pārējām metodēm.

Reply:
Ja es nemaldos tad tas ResponseEntity ir nepieciešams testiem, kuros prasa gan flight objektu, gan response status, tādēļ tika izmantots ResponseEntity